motionplan/motionPlanner.go

```diff
@@ -92,6 +180,12 @@ type node interface {
 	Q() []frame.Input
 }
 
+type planReturn interface {
+	// return the steps in Input form
+	ToInputs() [][]frame.Input
+	Err() error
```

Comment:
[q] where would these be called outside of this package?

Reply:
It wouldn't; I took this private.